modules/profiling-ext/report/libs/Chart.min.js

```diff
@@ -0,0 +1,10 @@
+/*!
+ * Chart.js
+ * http://chartjs.org/
+ * Version: 2.7.3
+ *
+ * Copyright 2018 Chart.js Contributors
+ * Released under the MIT license
```

Comment:
Are we ok to use mix of MIT and Apache license?

Reply:
It's ok, I have mentioned these components in the LICENSE file.